package.json

```diff
@@ -0,0 +1,5 @@
+{
+	"devDependencies": {
```

Comment:
Tabs?

Reply:
The file was created by `lerna`

Reply:
Ohh.. @maximuk then you should spank `lerna` =)